src/pathfinding_service/claim_fees.py

```diff
@@ -30,10 +31,10 @@
     help="Current RDN/ETH price, used to check claimed amount > transaction cost",
 )
 @click.option(
-    "--expires-within",
-    default=4 * 60 * 24 * 7,  # one week
+    "--claimable-until-within",
+    default=60 * 60 * 24 * 7,  # one week
     type=click.IntRange(min=1),
-    help="Only IOUs which expire withing this number of blocks will be claimed",
+    help="Only IOUs which are claimable until the time range of now plus this number of seconds will be claimed",  # noqa: E501
```

Comment:
Within the Raiden codebase, the intended naming system so far was that points in time are named "expiration" and time ranges "timeout" (which was not applied correctly in every case). The naming in this change works well in many cases, but in this specific case, I find it hard to read and understand.

Reply:
That is very true. I made this change very "blindly" without much feelings. Do you have an idea what would sound better? Feel free to many suggestions.